src/tdastro/astro_utils/dustmap.py

```diff
@@ -144,3 +145,50 @@ def query(self, coords):
             The E(B-V) value or array of values.
         """
         return self.compute_ebv(coords.ra.deg, coords.dec.deg)
+
+
+class SFDMap(DustEBV):
+    """
+    A dustmap using the sfdmap2 package (https://github.com/AmpelAstro/sfdmap2)
+    """
+
+    def __init__(self, **kwargs):
```

Comment:
You will need to expand this to download the data files if they do not already exist. They should go into the `data/dustmaps` directory. The best way to handle this would be the pooch library. @OliviaLynn is looking at adding this library for passbands, so now might be a good time to incorporate it throughout TDAstro.